src/modules/nodes/nodeApi.ts

```diff
@@ -328,14 +328,16 @@ export const putNode = ({ taxonomyVersion, id, ...params }: PutNodeParams): Prom
 
 export interface PutResourcesPrimaryParams extends WithTaxonomyVersion {
   id: string;
+  recursive: boolean;
 }
 
 export const putResourcesPrimary = ({
   id,
+  recursive,
   taxonomyVersion,
 }: PutResourcesPrimaryParams): Promise<void> =>
   putAndResolve({
-    url: `${baseUrl}/${id}/makeResourcesPrimary`,
+    url: `${baseUrl}/${id}/makeResourcesPrimary?recursive=${recursive}`,
```

Comment:
Du burde sende inn params her istedenfor. Det ordner `?`-problematikk for oss automatisk.

Reply:
Fant ikkje umiddelbart kossen. Kan sjå på det en gong til.

Reply:
Det heter queryParams!

Reply:
Den er kun brukt på fetch og ikkje på put.